elements-harness/build.rs

```diff
@@ -16,10 +15,10 @@ fn main() {
             let stderr = String::from_utf8(output.stderr)
                 .unwrap_or_else(|_| String::from("Could not decode stderr"));
```

Comment:
That looks like you misunderstood what `unwrap_or_else` does? You are assigning the string "Could not decode stderr" to `stderr` here in case the `from_utf8` fails. Might be better to panic here, this is just a build-script anyway!

Sorry I missed this on the first pass!

Reply:
No worries. Fixed now.